open-company/open-company-web
# Profile refresh

Card: https://trello.com/c/AVoaAgtw

Abstract: https://share.goabstract.com/0e6f9ed2-ca96-4290-9db3-ffc2110fb322

To test desktop:
- click on the menu
- open the user profile
- change some of your data included the avatar
- save
- switch to notifications
- change them
- refresh
- [x] changed data still there? Good


To test mobile:
- click on the user icon top left
- [x] icon are aligned in the menu? Good
- [x] can't see Digest Settings and Invite People? Good
- open profile
- [x] can you change the user avatar? Good
- [x] can you change other infos? Good
- [x] can you change notifications settings? Good
- get out of the profile setup
- open the menu again (click on user avatar)
- [x] if you scroll can you NOT see the board behind the current view scrolling? Good
- click the X in top left corner
- [x] did the menu went away? Good
- [x] did  the board appeared? Good
